### PR TITLE
new blocks textures

### PR DESCRIPTION
**Pull request by ghost**
Ores , gem blocks

Credit : Hatsondogs for dried clay , mud bricks  item frame , lavawood ,soils
Stabo made pig iron block
Has made copper block &  cobalt block variants
Ron made rosegold 
firewood by OUN and plantkillable
Slimes muds made by Qbert

### CODE REVIEW

**Comment by futurenp** [comment]
"im okay with gngng etc"

But actually im ok with my brownstone textures being used in TIC

**Comment by futurenp** [comment]
im okay with TIC to use my work and im okay with MIT licenses
vik said it wrong so heres new one

File: src/main/resources/assets/tconstruct/models/block/cobalt_block.json [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+		"2": "tconstruct:block/cobalt_leftright",
+		"particle": "tconstruct:block/cobalt_topbottom"
+	},
+	"elements": [
+		{
+			"from": [0, 0, 0],
+			"to": [16, 16, 16],
+			"faces": {
+				"north": {"uv": [0, 0, 16, 16], "texture": "#1"},
+				"east": {"uv": [0, 0, 16, 16], "texture": "#2"},
```
Comment (KnightMiner) [inline src/main/resources/assets/tconstruct/models/block/cobalt_block.json:15]: Do not use numbers for texture names.

File: src/main/resources/assets/tconstruct/models/block/cobalt_block.json [hunk 2]
```diff
@@ -1,6 +1,54 @@
...
+			"scale": [0.25, 0.25, 0.25]
+		},
+		"gui": {
+			"rotation": [30, 225, 0],
+			"scale": [0.625, 0.625, 0.625]
+		},
+		"fixed": {
+			"scale": [0.5, 0.5, 0.5]
+		}
+	}
```
Comment (KnightMiner) [inline src/main/resources/assets/tconstruct/models/block/cobalt_block.json:53]: Entirely redundant to `block/block`, use model parents properly.

File: src/main/resources/assets/tconstruct/textures/block/firewood.png.mcmeta [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
-      {"index": 5, "time": 10},
-      {"index": 4, "time": 7},
-      {"index": 5, "time": 15},
-      {"index": 6, "time": 28},
-      {"index": 5, "time": 19}
+      {
+	"index": 0,
+	"time": 80
+      },
+      {
```
Comment (KnightMiner) [inline src/main/resources/assets/tconstruct/textures/block/firewood.png.mcmeta:9]: That is really ugly formatting on the .mcmeta file. See how it was formatted before? Do exactly that.

**Review by KnightMiner — CHANGES_REQUESTED** [review]
As always, your JSON submissions need work.

Blocks is a poor caregory for textures. The metal blocks should be combined with the PR with ingots, and dried bricks combined with the item, as in both cases any comments we have (such as regarding color) will be applicable to both.

Also, as previously stated, slimy mud is being removed, we do not need textures for it.

**Comment by KnightMiner** [comment]
Closing as vik has stopped communicating with us.

For the record, metal handled by #4260, slimy mud removed, dried clay moving to Natura, lavawood and dried clay are unknown status.

**Pull request closed without merging**